realtime_ai_character/audio/speech_to_text/whisper.py

```diff
@@ -14,7 +14,7 @@
 DEBUG = False
 logger = get_logger(__name__)
 config = types.SimpleNamespace(**{
-    'model': 'tiny',
+    'model': os.getenv("LOCAL_WHISPER_MODEL", "tiny"),
```

Comment:
looks good! Can you change the default to base now? We have switched to use base

Reply:
Sure! I've updated the settings to use 'base' as the default now.